tests/csm/rest/test_query_deployment.py

```diff
@@ -42,14 +45,23 @@ def setup_class(cls):
         cls.ha_obj = HAK8s()
         cls.deploy_lc_obj = ProvDeployK8sCortxLib()
         cls.csm_obj = csm_api_factory("rest")
-        cls.csm_conf = configmanager.get_config_wrapper(fpath="config/csm/test_rest_capacity.yaml")
+        cls.csm_conf = configmanager.get_config_wrapper(
+            fpath="config/csm/test_rest_query_deployment.yaml")
         cls.update_seconds = cls.csm_conf["update_seconds"]
+        cls.failed_pod = None
+        cls.kvalue = None
+        cls.deploy_list = cls.csm_obj.master.get_deployment_name(POD_NAME_PREFIX)
+        cls.random_string = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
```

Comment:
Can u move this random generation to each test as we want it should generate different values for different tests.

Reply:
Even though it is present in setup, it will generate random values for each test. This has been checked.

Reply:
No it will generate once per class.

Reply:
it will work with pytest, but for testrunner it will fail.

Reply:
while running multiple tests, issue was observed. Hence moved to individual tests.